jaseci_core/jaseci/api/actions_api.py

```diff
@@ -79,7 +79,7 @@ def actions_load_remote(self, url: str):
         return {"success": success}
 
     @Interface.admin_api(cli_args=["mod"])
-    def actions_load_module(self, mod: str):
+    def actions_load_module(self, mod: str, ctx: dict = {}):
```

Comment:
This should also be supported with actions load local and remote. The corresponding functions for those two are in this file too.